en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md

```diff
@@ -211,7 +238,7 @@ Given below are the main steps your will follow when you deploy integration solu
 When the integration is successfully deployed, it should create the `hello-world` integration, `hello-world-deployment`, `hello-world-service`, and `ei-operator-ingress` as follows:
 
 !!! Tip
-    The `ei-operator-ingress` will not be created if you have [disabled the ingress controller](#disable-ingress-controller).
+    The `ei-operator-ingress` will not be created if you have [disabled the ingress controller](#Disable-ingress-controller-creation).
```

Comment:
ei-operator-ingress?
Do we still use 'ei'?

Reply:
This needs a code change. Shall we take it in a different PR?

Reply:
+1